src/app/task-list/task-list.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="col-lg-3 col-lg-offset-4">
     <form [formGroup]="taskForm" (submit)="addTask(task.value)">
       <div class="input-group">
-        <input type="text" name="task" formControlName="task" placeholder="typeYourTaskHere" class="form-control" #task>
+        <input type="text" name="task" formControlName="task" placeholder="Type Your Task Here" class="form-control" #task>
```

Comment:
Please use lowercase for all except the first word

Reply:
Fix is done.